### PR TITLE
W-13469739 Google BigQuery Correcting descriptions

### PR DESCRIPTION
**Pull request by valkyrie69**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: google-bigquery/1.1/modules/ROOT/pages/google-bigquery-connector-reference.adoc [hunk 1]
```diff
@@ -465,9 +465,7 @@ Connector does not stream the request content. |  |
 === Get Query Results
 `<bigquery:get-query-results>`
 
-
-Get results of a query. This operation makes an HTTP GET request to the /files/{fileId}/comments endpoint.
-
+This operation makes an HTTP GET request to the projects/{projectId}/queries/{jobId} endpoint.
```
Comment (sathya0) [inline google-bigquery/1.1/modules/ROOT/pages/google-bigquery-connector-reference.adoc]: should we add the operation description back in?

Comment (sathya0) [inline google-bigquery/1.1/modules/ROOT/pages/google-bigquery-connector-reference.adoc]:
```suggestion
This operation makes an HTTP GET request to the /projects/{projectId}/queries/{jobId} endpoint.
```

File: google-bigquery/1.1/modules/ROOT/pages/google-bigquery-connector-reference.adoc [hunk 2]
```diff
@@ -632,7 +630,7 @@ Connector does not stream the request content. |  |
 `<bigquery:list-jobs>`
 
 
-Lists jobs. This operation makes an HTTP GET request to the /files/{fileId}/comments endpoint.
+Lists jobs. This operation makes an HTTP GET request to the projects/{projectId}/jobs endpoint.
```
Comment (sathya0) [inline google-bigquery/1.1/modules/ROOT/pages/google-bigquery-connector-reference.adoc]:
```suggestion
Lists jobs. This operation makes an HTTP GET request to the /projects/{projectId}/jobs endpoint.
```

File: google-bigquery/1.1/modules/ROOT/pages/google-bigquery-connector-reference.adoc [hunk 3]
```diff
@@ -720,7 +718,7 @@ Connector does not stream the request content. |  |
 `<bigquery:query>`
 
 
-Executes a query. This operation makes an HTTP GET request to the /files/{fileId}/comments endpoint.
+This operation makes an HTTP POST request to the /projects/{projectId}/queries endpoint.
```
Comment (sathya0) [inline google-bigquery/1.1/modules/ROOT/pages/google-bigquery-connector-reference.adoc]: should we add the operation description back in?

File: google-bigquery/1.0/modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -10,7 +10,7 @@ For information about compatibility and fixed issues, refer to the Google BigQue
 
 To use this connector, you must be familiar with:
 
-* The Google BigQuery Connector’s API
+* The Google BigQuery API
 * Anypoint Connectors
```
Comment (sathya0) [inline google-bigquery/1.0/modules/ROOT/pages/index.adoc:14]: should we add those reference changes to 1.0 too?

**Review by sathya0 — DISMISSED** [review]

File: google-bigquery/1.1/modules/ROOT/pages/google-bigquery-connector-reference.adoc (hunk 1 above)
Comment (valkyrie69) [reply google-bigquery/1.1/modules/ROOT/pages/google-bigquery-connector-reference.adoc]: Nah, I think it's self-evident. It didn't add much.

File: google-bigquery/1.0/modules/ROOT/pages/index.adoc (hunk 4 above)
Comment (valkyrie69) [reply google-bigquery/1.0/modules/ROOT/pages/index.adoc:14]: @sathya0 Yes, it's in this PR.

**Review by sathya0 — DISMISSED** [review]

**Review by sathya0 — APPROVED** [review]

**Pull request merged**